src/js/stores/MesosStateStore.js

```diff
@@ -300,6 +314,7 @@ class MesosStateStore extends GetSetBaseStore {
       if (name === "marathon") {
         return tasks
           .concat(completed_tasks)
+          .concat(unreachable_tasks)
```

Comment:
why not re-use the first `concat(completed_tasks, unreachable_tasks)` as one operation?